core/src/main/java/org/sonarsource/sonarlint/core/BindingSuggestionProviderImpl.java

```diff
@@ -117,22 +123,33 @@ public void connectionAdded(ConnectionConfigurationAddedEvent event) {
     }
   }
 
+  @Override
+  public CompletableFuture<SuggestBindingResponse> suggestBinding(GetBindingSuggestionParams params) {
+    return CompletableFuture.supplyAsync(() -> {
+      var suggestions = computeBindingSuggestions(Set.of(params.getConfigScopeId()), Set.of(params.getConnectionId()));
+      return new SuggestBindingResponse(suggestions);
+    }, executorService);
+  }
+
   private void queueBindingSuggestionComputation(Set<String> configScopeIds, Set<String> candidateConnectionIds) {
     executorService.submit(() -> {
-      try {
-        if (enabled.get()) {
-          bindingSuggestionComputation(configScopeIds, candidateConnectionIds);
-        } else {
-          LOG.debug("Skipping binding suggestion computation as it is disabled");
-        }
-      } catch (InterruptedException e) {
```

Comment:
Out of curiosity, why have you removed this?

Reply:
The log is in the [queueBindingSuggestionComputation](https://github.com/SonarSource/sonarlint-core/blob/feature/vs/SLCORE-477_implement_option_to_query_binding_suggestions/core/src/main/java/org/sonarsource/sonarlint/core/BindingSuggestionProviderImpl.java#L139) method now.

Reply:
I was more asking for the try catch